llama_index/indices/knowledge_graph/base.py

```diff
@@ -62,7 +63,10 @@ def __init__(
                 max_knowledge_triplets=self.max_triplets_per_chunk
             )
         )
-
+        # allows for direct triplet initialization
+        if triplets is not None:
+            for triplet in triplets:
+                self.upsert_triplet(triplet)
```

Comment:
Have you tested that this works? 

AFAIK the index_struct hasn't been initialized here yet, so this might run into an error?